shared/views/RoomCardView.js

```diff
@@ -70,7 +70,7 @@ class RoomCardView extends TemplateView {
           },
           [aliasOrRoomId]
         ),
-        t.p({ className: 'RoomCardView_topic' }, [vm.topic || '']),
+        t.p({ className: 'RoomCardView_topic', title: vm.topic || null }, [vm.topic || '']),
```

Comment:
Verified that `null` will cause no `title` attribute to be added to the element.

`undefined` will show up as `title="undefined"`